lib/components/collapse.vue

```diff
@@ -43,10 +43,9 @@
         },
 
         watch: {
-            visible(newVal, oldVal){
-                if (newVal !== this.show) {
-                    this.toggle()
-                }
+            visible(newVal) {
+                this.show = newVal;
```

Comment:
- The watcher will only be called once the value of `visible` has changed
- If we don't want to emit unnecessary events, we check against `show` for a change
- At this point we're performing the same actions in the toggle method

Why not leave it calling toggle to keep the logic consolidated?